config/initializers/google_cloud_trace.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Fake google cloud trace in_span method so we don't have to check our environment
```

Comment:
Is it worth sticking this in stackdriver.rb like the rest of the trace initialization? Also, I'm confused that we don't have to do this on the tidelift side?

Reply:
Hmm yea I’ll take a look at doing that in a follow up.